docs/sphinx/user_guide/run.rst

```diff
@@ -138,3 +138,111 @@ was not possible for them to co-exist in the same executable as CUDA
 variants, for example. In the future, the build system may be reworked so 
 that the OpenMP target variants can be run from the same executable as the 
 other variants.
+
+============================
+Additional Caliper Use Cases
+============================
+
+If you specified building with Caliper `-DRAJA_PERFSUITE_USE_CALIPER=On`,
+for the most part the generation of Caliper .cali files are automated.
+However, there are a couple of other supported use cases.
+
+Collecting PAPI topdown statistics on Intel Architectures
+---------------------------------------------------------
+
+On Intel systems, you can collect topdown PAPI counter statistics by using a command line switch
+
+``--add-to-spot-config, -atsc <string> [Default is none]``
+
+appends additional parameters to the built-in Caliper spot config
+
+Example to include some PAPI counters (Intel arch)
+
+``-atsc topdown.all``
+
+Caliper's topdown service generates derived metrics from raw PAPI counters; a hierarchy of metrics to identify bottlenecks in out-of-order processors. This is based on an an approach described in Ahmad Yasin's paper *A Top-Down Method for Performance Analysis and Counters Architecture*. The toplevel of the hierarchy has a reliable set of four derived metrics or starting weights (sum to 1.0) which include:
+
+1. Frontend Bound: stalls attributed to the front end which is responsible for fetching and decoding program code.    
+2. Bad Speculation: fraction of the workload that is affected by incorrect execution paths, i.e. branch misprediction penalties
+3. Retiring: Increases in this category reflects overall Instructions Per Cycle (IPC) fraction which is good in general. However, a large retiring fraction for non-vectorized code could also be a hint to the user to vectorize their code (see Yasin's paper) 
+4. Backend Bound: Memory Bound where execution stalls are related to the memory subsystem, or Core Bound where execution unit occupancy is sub-optimal lowering IPC (more compiler dependent)
+
+> Backend Bound = 1 - (Frontend Bound + Bad Speculation + Retiring)
+
+Caveats: 
+
+1. When collecting PAPI data in this way you'll be limited to running only one variant, since Caliper maintains only a single PAPI context.
+2. Small kernels should be run at large problem sizes to minimize anomalous readings 
+3. Measured values are only relevant for the innermost level of the Caliper tree hierarchy, i.e. kernel.tuning under investigation
+4. Some lower level derived quantities may still appear anomalous with negative values. Collecting raw counters can help identify the discrepancy
+
+``-atsc topdown-counters.all``
+
+A Hatchet based snippet to fetch topdown.toplevel metric values at the leaf (Kernel.Tuning) nodes across several files using -atsc topdown.toplevel::
+
+ def load_toplevel(cr):
```

Comment:
Similar comment about linking in content from the script file to prevent docs and scripts getting out of sync.